examples/03-Running-on-multiple-GPUs-or-on-CPU.ipynb

```diff
@@ -0,0 +1,742 @@
+{
```

Comment:
<p>The title is advanced workflow?</p><p> </p><p>We can add something like:</p><p>"NVTabular supports to change easily between multi-GPU, single GPU and CPU requires by changing only few lines."</p><p>"A common use-case is to develop locally with CPU only and then deploy the NVTabular workflow in the cloud on a multi-GPU cluster"</p><p> </p><p>We should add a comment, why we do not look at single GPU - because it is the default behavior?</p><p>Can you check with Ben, what is the default behavior? </p><p>Is it GPU </p><p>or </p><p>is it GPU, if GPU is available - otherwise, CPU?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Made the changes. Could you please share a bit more details on what you'd like me to follow up with Ben on? What I have experienced is that if GPU is available, it will be used by default, but when it is not available, CPU is used. Please let me know if this is what you are referring to or something else. Thank you!</p>

Reply:
The default operation is like what @radekosmulski described: we will use the GPU if is available, and CPU if not. You can force CPU operation on a GPU system though by setting the 'cpu' parameter on the Dataset class